toolkits/global/packages/global-corporate-footer/view/corporate-level.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="c-corporate-footer">
+    <div class="u-container">
```

Comment:
It is preferable not to rely on utility classes as it introduces a point of failure - we rely on the app including the correct utility classes. Sometimes it is unavoidable, but in this case it would be fairly simple to create a `c-corporate-footer__container` class and use the [`@mixin`](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/default/scss/30-mixins/container.scss)

```scss
.c-corporate-footer__container {
	@include u-container();
}
```

you can then simplify the readme and the demo so that there is no reliance on the utility.